### PR TITLE
chore (PURCHASE-2814): add test to the conversation component

### PR DESCRIPTION
**Pull request by copperkraft**
Partially solves https://artsyproduct.atlassian.net/browse/PURCHASE-2814

Move Item tests to a separate file, add tests to the Conversation component

### CODE REVIEW

File: src/v2/Apps/Conversation/Components/__tests__/Item.jest.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import { mount } from "enzyme"
+import React from "react"
+import { Item } from "../Item"
+import { Conversation_conversation } from "v2/__generated__/Conversation_conversation.graphql"
+
+describe("Item", () => {
```
Comment (xtina-starr) [inline src/v2/Apps/Conversation/Components/__tests__/Item.jest.tsx:6]: Good decision to extract this out into its own test file 🙌🏿

File: src/v2/Apps/Conversation/Components/Conversation.tsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ const Conversation: React.FC<ConversationProps> = props => {
   const bottomOfMessageContainer = useRef<HTMLElement>(null)
 
   const liveArtwork = conversation?.items?.[0]?.liveArtwork
-  const artwork = liveArtwork?.__typename === "Artwork" && liveArtwork
+  const artwork = liveArtwork?.__typename === "Artwork" ? liveArtwork : null
```
Comment (xtina-starr) [inline src/v2/Apps/Conversation/Components/Conversation.tsx:43]: 👏  I've done the same for this in the eigen implementation.

File: src/v2/Apps/Conversation/Components/__tests__/Item.jest.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  describe("when inquiry item is an artwork", () => {
+    // @ts-expect-error STRICT_NULL_CHECK
```
Comment (damassi) [inline src/v2/Apps/Conversation/Components/__tests__/Item.jest.tsx:8]: These STRICT_NULL_CHECKS were added by a codemod when migrating to strict type checking and should be avoided

File: src/v2/Apps/Conversation/Components/__tests__/Conversation.jest.tsx [hunk 4]
```diff
@@ -1,89 +1,134 @@
...
-          __typename: "Money",
-          display: "$2000",
-        },
+jest.unmock("react-relay")
+
+const render = (
+  node: ConversationPaginationTestQueryRawResponse["node"],
+  user: User
+) =>
+  renderRelayTree({
```
Comment (damassi) [inline src/v2/Apps/Conversation/Components/__tests__/Conversation.jest.tsx:24]: Is it possible to use our modern relay test tools and patterns? See the [Best Practices](https://github.com/artsy/force/blob/master/docs/best_practices.md#write-unit-tests-for-new-components) doc for more info.

**Review by xtina-starr — APPROVED** [review]
This is a good step forward that at least allows us to get some of the baseline functionality covered. Perhaps we can pair together later this week to iterate on this to use the newer pattern. I think that would require us to refactor some of the Conversation component to make it more easy to test.

**Comment by damassi** [comment]
Ah sorry @xtina-starr - not sure if you saw my comments before merging -- left a few.

File: src/v2/Apps/Conversation/Components/__tests__/Conversation.jest.tsx (hunk 4 above)
Comment (xtina-starr) [reply src/v2/Apps/Conversation/Components/__tests__/Conversation.jest.tsx:24]: @damassi yeah we chatted about that. I think we need to refactor this component in order to do so. @copperkraft has a work in progress PR for [this](https://github.com/artsy/force/pull/8122). I'm creating a ticket for us to do that this week. I felt like the outdated test pattern was better than no coverage at all.

Comment (damassi) [reply src/v2/Apps/Conversation/Components/__tests__/Conversation.jest.tsx:24]: Cool, that's what I figured 👍 np!

**Pull request merged**